devtools_m2/docker-compose.base.yml

```diff
@@ -14,6 +14,7 @@ services:
       - type: bind
         source: './..'
         target: /var/www/html/magento/app/code/Drip/Connect
+        consistency: cached
```

Comment:
This makes the host (your local machine, not the docker container) the authority on what should appear in `.../Drip/Connect`.  It reduces over head in the docker container, which [theoretically] helps keep the PHP+Xdebug environment performant.